graphql/resolvers.go

```diff
@@ -1837,6 +1838,56 @@ func (r *taskResolver) LatestExecution(ctx context.Context, obj *restModel.APITa
 	return task.GetLatestExecution(*obj.Id)
 }
 
+func (r *queryResolver) SearchReturnInfo(ctx context.Context, taskId string, exec string) (*thirdparty.SearchReturnInfo, error) {
```

Comment:
is there a way to consolidate this function with the old UI's [search function](https://github.com/evergreen-ci/evergreen/blob/e8a4f20b335fe6efbfe1a0ec9086785003067445/service/ui_plugin_build_baron.go#L169)?

Reply:
It's essentially the same code. Eventually everything in the service package will be deleted once the old UI is deprecated.